### ISSUE TITLE
With this content we get `unable to tokenize`

### ISSUE THREAD

**Issue by atimmer** [Yoast/YoastSEO.js#1181]
This content causes an `unable to tokenize` issue:

```
<a name='top'></a>hreflang tags are a technical solution for sites that have similar content in multiple languages. A site owner wants search engines to point people to the most "fitting" language. Say a user is Dutch, the page that ranks is English, but there's also a Dutch version. You would want Google to show the Dutch page in the search results for that user. This is the type of problem hreflang was designed to solve.

In this (<em>very</em> long) article we'll discuss:
<ul>
 	<li>What <a href='#hreflang-meant-for'>hreflang is for</a>.</li>
 	<li>What the <a href='#hreflang-seo-benefit'>SEO benefit of hreflang</a> is.</li>
 	<li><a href='#what-is-hreflang'>What hreflang <em>is</em></a><em>.</em></li>
 	<li><a href='#what-hreflang-accomplishes'>What hreflang accomplishes</a>.</li>
 	<li>Whether or not <a href='#should-you-use-hreflang'>you should use hreflang</a>.</li>
 	<li>Which <a href='#hreflang-architecture'>architectural decisions</a> you should make.</li>
 	<li>The <a href='#hreflang-technical-basics'>basic technical ideas about hreflang</a> you should know.</li>
 	<li><a href='#hreflang-implementation-choices'>Implementation decisions</a>  you should make.</li>
 	<li>Which <a href='#hreflang-technical-2'>other technical things</a> we need to account for, like:
<ul>
 	<li><a href='#x-default'><code>x-default</code></a></li>
 	<li><a href='#hreflang-canonical'>how <code>rel="canonical"</code> and hreflang</a> work together.</li>
</ul>
</li>
 	<li>Which <a href='#hreflang-tools'>tools</a> to use when developing and testing your  implementation.</li>
 	<li>How to <a href='#hreflang-process'>keep your implementation working</a> once you've set it up.</li>
</ul>
hreflang tags are among the hardest specs I've ever seen come out of a search engine. Doing it right is tough and takes time. This guide will try and prevent you from falling into common traps. Be sure to read it thoroughly if you're embarking on an hreflang project.<section class='alignright extra show-off'><p><a class='link--naked link--cta' href='https://yoast.com/hire-us/yoast-seo-care/'>Let our SEO experts analyze and optimize your site: Get Yoast SEO Care! »</a></p><a class='hide-on-mobile' href='https://yoast.com/hire-us/yoast-seo-care/'><img src='https://yoast-mercury.s3.amazonaws.com/uploads/2016/12/Posts_SEO_Care_banner-2.png' alt='Yoast SEO Care'></img></a><a class='button default disabled' href='javascript:void();'>Temporarily sold out</a><a href='https://yoast.com/hire-us/yoast-seo-care/' class='button dimmed flat'><i class='fa fa-info-circle'></i> Info</a></section>
<h2 id='hreflang-meant-for'>What are hreflang tags meant for?</h2>
<strong>hreflang tags</strong> are a method to mark up pages that are similar in meaning but aimed at different languages and/or regions. You can use this for three types of variations:
<ul>
 	<li>Content with regional variations like <code>en-us</code> and <code>en-gb</code>.</li>
 	<li>Content in different languages like <code>en</code>, <code>de</code> and <code>fr</code>.</li>
 	<li>A combination of different languages and regional variations.</li>
</ul>
You can use hreflang tags to target different markets that use the same language. This is a fairly common use case. Using hreflang tags you can differentiate between the US and the UK, or between Germany and Austria.

<img class='alignnone size-full wp-image-680677' src='https://yoast-mercury.s3.amazonaws.com/uploads/2016/04/9_Use_Hreflang_FI-1.png' alt='9_Use_Hreflang_FI (1)' width='1200' height='628'></img>
<h2 id='hreflang-seo-benefit'>What's the SEO benefit of hreflang?</h2>
So why are we even talking about hreflang? What is the SEO benefit? There are two main reasons, from an SEO point of view, why you should implement it.

First and foremost, if you have a version of a page that you have optimized for the users language and location, you want them to land on that page. Having the right language and location dependent information improves their user experience and thus leads to fewer people bouncing back to the search results. Fewer people bouncing back to the search results leads to higher rankings.

The second reason is that hreflang prevents a <a href='https://yoast.com/duplicate-content/' target='_blank'>duplicate content problem</a>. You might have the same content in English on different URLs aimed at the UK, the US and Australia. The difference on these pages might be as small as a change in prices and currency. Google might not understand on its own what you're trying to do and see it as duplicate content. With hreflang you make it very clear to the search engine that it's (almost) the same content, just optimized for different people.
<h2 id='what-is-hreflang'>What <em>is</em> hreflang?</h2>
hreflang is code, which you can show to search engines in three different ways, more on that below. With this code you specify all the different URLs on your site(s) that have the same content. These URLs can have the same content in a different language, or the same language but targeted at a different region.
<h2 id='#what-hreflang-accomplishes'>What does hreflang accomplish?</h2>
<section class='alignright extra' id=''><h4><i class='fa fa-question-circle'></i> Who supports hreflang?</h4><p>hreflang is supported by Google and Yandex. Bing doesn't have an equivalent, but does support <a href='https://blogs.bing.com/webmaster/2011/03/01/how-to-tell-bing-your-websites-country-and-language/' target='_blank'>language meta tags</a>.</p>
</section>

In a complete hreflang implementation, every URL specifies which other variations are available. When a user searches, Google goes through the following process:
<ol>
 	<li>it determines that it wants to rank a URL;</li>
 	<li>it checks whether that URL has hreflang annotations;</li>
 	<li>it presents the searcher with the results with the most appropriate URL for that user.</li>
</ol>
The users current location <em>and</em> his language settings determine the most appropriate URL. A user can have multiple languages in his browser's settings. I, for instance, have Dutch, English and German in there. The order in which these languages appear in my settings determines the most appropriate language.
<h2 id='should-you-use-hreflang'>Should you use hreflang?</h2>
<section class='alignright extra' id=''><h4><i class='fa fa-info-circle'></i> Tip: homepage first!</h4><p>If you're not sure on whether you want to implement hreflang on your entire site, start with your homepage! People searching for your brand will get the right page. This is a lot easier to implement and it will "catch" a large part of your traffic.</p>
</section>

Based on what we've learned on what hreflang is and how it works, we can determine if you should use it. You should use it if:
<ul>
 	<li>you have the same content in multiple languages;</li>
 	<li>you have content aimed at different geographic regions, but in the same language.</li>
</ul>
Whether the content you have resides on one domain or multiple domains does not matter. You can link variations within the same domain but can also link between domains.<img class='noborder alignnone wp-image-676677 size-full' src='https://yoast-mercury.s3.amazonaws.com/uploads/2016/03/Hreflang_mistakes_FI.png' alt='hreflang: the ultimate guide' width='1200' height='628'></img>
<h2 id='hreflang-architecture'>Architectural implementation choices</h2>
One thing is very important when implementing hreflang: don't be too specific! Let's say you have three types of pages:
<ul>
 	<li>German</li>
 	<li>German, specifically aimed at Austria</li>
 	<li>German, specifically aimed at Switzerland</li>
</ul>
You could choose to implement them using three hreflang attributes like this:
<ul>
 	<li><code>de-de</code> targeting German speakers in Germany</li>
 	<li><code>de-at</code> targeting German speakers in Austria</li>
 	<li><code>de-ch</code> targeting German speakers in Switzerland</li>
</ul>
However, which of these three results should Google show to someone searching in German in Belgium? The first page would probably be the best. To make sure that every German searching user who does not match either <code>de-at</code> or <code>de-ch</code> gets that one, change that hreflang attribute to just <code>de</code>. Specifying <em>just</em> the language is in many cases a smart thing to do.

It's good to know that when you create sets of links like this, the most specific one wins. The order in which the search engines sees the links doesn't matter, it'll always try to match from most specific to least specific.
<h2 id='hreflang-technical-basics'>Technical implementation basics</h2>
Regardless of which type of implementation you choose (more on that below), there are three basic rules.
<h3>1. Valid hreflang attributes</h3>
The hreflang attribute needs to contain a value that consists of the language, optionally combined with a region. The language attribute needs to be in <a href='http://en.wikipedia.org/wiki/List_of_ISO_639-1_codes'>ISO 639-1 format</a> (a two letter code).

<section class='alignright extra' id=''><h4><i class='fa fa-exclamation-circle'></i> Wrong region codes</h4><p>Google can deal with some of the common mistakes with region codes, although you shouldn't take any chances. For instance, it can deal with <code>en-uk</code> just as well as with the "right" <code>en-gb</code>. However, <code>en-eu</code> does <em>not</em> work, as <code>eu</code> doesn't define a country.</p>
</section>

The region is optional and should be in ISO 3166-1 Alpha 2 format, more specifically, it should be an officially assigned element. This means you need to use <a href='https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements' target='_blank'>this list</a> for verification. This is where things often go wrong: using the wrong region code is a very common problem. Use the linked lists on Wikipedia to verify that you're using the right region and language codes.
<h3>2. Return links</h3>
The second basic rule is about return links. Regardless of your type of implementation, each URL needs return links to every other URL, note that it should point at the canonical versions, <a href='#hreflang-canonical'>more on that below</a>. The more languages you have the more you might be tempted to limit those return links: don't. If you have 80 languages, you'll have hreflang links for 80 URLs. There's no getting around that.<span style='font-weight: 400;'><section class='alignright extra show-off'><p><a class='link--naked link--cta' href='https://yoast.com/academy/course/technical-seo-1-training/'>Become a technical SEO expert with our Technical SEO 1 training! »</a></p><a class='hide-on-mobile' href='https://yoast.com/academy/course/technical-seo-1-training/'><img src='https://yoast-mercury.s3.amazonaws.com/uploads/2017/02/Technical_SEO_1_banner.png' alt='Technical SEO 1 training'></img></a><a class='button default' rel='nofollow' href='https://my.yoast.com/checkout/?edd_action=add_to_cart&download_id=1288999&yst_action_edd=add_to_cart'><span class='yoast-currency yoast-currency__USD'>$ 199</span><span class='yoast-currency yoast-currency__EUR hidden'>€ 199</span> - Buy now »</a><a href='https://yoast.com/academy/course/technical-seo-1-training/' class='button dimmed flat'><i class='fa fa-info-circle'></i> Info</a></section></span><span style='font-weight: 400;'>
</span>
<h3>3. hreflang link to self</h3>
The third and final basic rule is about self-links. Just like those return links might feel weird at some point, the hreflang link to the current page feels weird for some developers. It's required though and not having it will mean your implementation will not work.
<h2 id='hreflang-implementation-choices'>Technical implementation choices</h2>
There are three ways to implement hreflang: using link elements in the <code><head></code>, using HTTP headers or using an XML sitemap. Each has its uses, so we'll discuss them and explain why you should choose any of these.
<h3>1. HTML hreflang link elements in your <head></h3>
The first method to implement hreflang we'll discuss is HTML hreflang link elements. To implement hreflang using header link elements, you add code like this to the <code><head></code> section of every page:
<pre><link rel="alternate" href="http://example.com/" 
      hreflang="en" />
<link rel="alternate" href="http://example.com/en-gb/" 
      hreflang="en-gb" />
<link rel="alternate" href="http://example.com/en-au/" 
      hreflang="en-au" />
</pre>
As every variation needs to link to every other variation, these implementations can become quite big and lead to performance issues. If you have 20 languages, choosing HTML link elements would mean adding 20 link elements as shown above to every page. This means adding 1.5KB on every page load, that no user will ever use, but will have to download. On top of that, your CMS will have to do several database calls to generate all these links. This markup is purely meant for search engines. That's why I would not recommend this for larger sites, as it adds far too much, unneeded, overhead.
<h3>2. hreflang HTTP headers</h3>
The second method of implementing hreflang is through HTTP headers. HTTP headers are the solution for all your PDFs and other non-HTML content you might want to optimize. Link elements work nicely for HTML documents, but not for other types of content as you can't include them. That's where HTTP headers come in. They should look like this:
<pre>Link: <http://es.example.com/document.pdf>; 
rel="alternate"; hreflang="es", 
<http://en.example.com/document.pdf>; 
rel="alternate"; hreflang="en", 
<http://de.example.com/document.pdf>; 
rel="alternate"; hreflang="de"
</pre>
The problem with having a lot of HTTP headers is similar to the problem with link elements in your <head>: it adds a lot of overhead to every request.
<h3>3. An XML sitemap hreflang implementation</h3>
The third option to implement hreflang is using XML sitemap markup. It uses the xhtml:link attribute in XML sitemaps to add the annotation to every URL. It works very much in the same way as you would in a page's <head> with <link> elements. If you thought link elements were verbose, the XML sitemap implementation is even worse. This is the markup needed for just one URL with two other languages:
<pre><url>
  <loc>http://www.example.com/uk/</loc> 
  <xhtml:link rel="alternate" hreflang="en" 
     href="http://www.example.com/" /> 
  <xhtml:link rel="alternate" hreflang="en-au" 
     href="http://www.example.com/au/" /> 
  <xhtml:link rel="alternate" hreflang="en-gb" 
     href="http://www.example.com/uk/" />
</url>
</pre>
You can see it has a self-referencing URL as the third URL, specifying the specific URL is meant for <code>en-gb</code>, and it specifies two other languages. Now, both other URLs would need to be in the sitemap too, looking like this:
<pre><url>
  <loc><span style='color: #ff0000;'>http://www.example.com/</span></loc> 
  <xhtml:link rel="alternate" hreflang="en" 
     href="http://www.example.com/" /> 
  <xhtml:link rel="alternate" hreflang="en-au" 
     href="http://www.example.com/au/" /> 
  <xhtml:link rel="alternate" hreflang="en-gb" 
     href="http://www.example.com/uk/" />
</url>
<url>
  <loc><span style='color: #ff0000;'>http://www.example.com/au/</span></loc> 
  <xhtml:link rel="alternate" hreflang="en" 
     href="http://www.example.com/" /> 
  <xhtml:link rel="alternate" hreflang="en-au" 
     href="http://www.example.com/au/" /> 
  <xhtml:link rel="alternate" hreflang="en-gb" 
     href="http://www.example.com/uk/" />
</url></pre>
As you can see, basically we're only changing the URLs within the <code><loc></code> element, as everything else needs to be the same. This way, each URL has a self-referencing hreflang attribute, and return links to the appropriate other URLs.

XML sitemap markup like this is very verbose: a lot of output is needed to do this for a lot of URLs. The benefit of an XML sitemap implementation is simple: your normal users won't be bothered with this markup. This has the benefit of not adding extra page weight and it doesn't require a lot of database calls on page load to generate this markup.

Another benefit of adding hreflang through the XML sitemap is that it's usually a lot easier to change an XML sitemap than to change all the pages on a site. No need to go through large approval processes and maybe you can even get direct access to the XML sitemap file.
<h2 id='hreflang-technical-2'>Other technical aspects of an hreflang implementation</h2>
We're going to assume that you've decided which type of technical implementation you're going to choose. There are a couple of other technical specificities you should know about before you start implementing hreflang.
<h3 id='x-default'>hreflang x-default</h3>
There is a special hreflang attribute value that's called <code>x-default</code>. The <code>x-default</code> value specifies where a user should be sent if none of the languages you've specified in your other hreflang links match their browser settings. In a link element it looks like this:
<pre><link rel="alternate" href="http://example.com/" 
  hreflang="x-default" /></pre>
When it <a href='https://webmasters.googleblog.com/2013/04/x-default-hreflang-for-international-pages.html' target='_blank'>was introduced</a>, it was explained as being for "international landing pages", ie pages where you redirect users based on their location. However, it can basically be described as the final "catch-all" of all the hreflang statements. If the users location and language didn't match anything else, that's where they will be sent.

In the German example we <a href='#hreflang-architecture'>mentioned above</a>, a user searching in English still wouldn't have a "fitting" URL. That's one of the cases where <code>x-default</code> comes into play. You'd add a fourth link to the markup, and end up with these 4:
<ul>
 	<li><code>de</code></li>
 	<li><code>de-at</code></li>
 	<li><code>de-ch</code></li>
 	<li><code>x-default</code></li>
</ul>
In this case, the <code>x-default</code> link would point to the <em>same</em> URL as the <code>de</code> one. We would not encourage you to remove the <code>de</code> link though, even though technically that would create exactly the same result. In the long run it's usually better to have both as it specifies what language that de page is in and makes the code easier to read.
<h3 id='hreflang-canonical'>hreflang and rel=canonical</h3>
<section class='alignright extra' id=''><h4><i class='fa fa-question-circle'></i> rel="canonical"</h4><p>If you don't know what <code>rel="canonical"</code> is, read <a href='https://yoast.com/rel-canonical/'>this article</a>!</p>
</section>

<code>rel="alternate" hreflang="x"</code>markup and <code>rel="canonical"</code> can and should be used together. Every language should have a <code>rel="canonical"</code> link pointing to itself. In the first example, this would look like this, assuming that we're on the example.com homepage:
<pre><link rel="canonical" href="http://example.com/">
<link rel="alternate" href="http://example.com/" 
      hreflang="en" />
<link rel="alternate" href="http://example.com/en-gb/" 
      hreflang="en-gb" />
<link rel="alternate" href="http://example.com/en-au/" 
      hreflang="en-au" />
</pre>
If we were on the <code>en-gb</code> page, not all that much would change other than the canonical:
<pre><link rel="canonical" href="<span style='color: #ff0000;'>http://example.com/en-gb/</span>">
<link rel="alternate" href="http://example.com/" 
      hreflang="en" />
<link rel="alternate" href="http://example.com/en-gb/" 
      hreflang="en-gb" />
<link rel="alternate" href="http://example.com/en-au/" 
      hreflang="en-au" />
</pre>
Don't make the mistake of setting the canonical on the <code>en-gb</code> page to <code>http://example.com/</code>, as this breaks the implementation. It's very important that the hreflang links point to the canonical version of each URL. These systems should work hand in hand!
<h2 id='hreflang-tools'>Useful tools when implementing hreflang</h2>
If you've come this far, you'll probably be thinking "wow this is hard", I know I thought that while learning about the topic. Luckily, there are quite a few tools available for people who dare to start implementing hreflang.
<h3>hreflang tag generator</h3>
<section class='alignright extra' id=''><p><a href='http://www.aleydasolis.com/en/international-seo-tools/hreflang-tags-generator/' target='_blank' rel='attachment wp-att-675728'><img class='alignnone size-large wp-image-675728' src='https://yoast-mercury.s3.amazonaws.com/uploads/2016/03/Hreflang-Tag-Generator-Tool-by-aleyda-600x265.png' alt='hreflang tag generator tool by Aleyda' width='600' height='265'></img></a></p>
</section>

Aleyda Solis, who has written quite a lot about this topic too, has created a very useful <a href='http://www.aleydasolis.com/en/international-seo-tools/hreflang-tags-generator/' target='_blank'>hreflang tag generator</a> that helps you generate link elements. Even when you're not choosing for a link element implementation, this can be useful to create some example code.
<h3>hreflang XML sitemap generator</h3>
The Media Flow have created an hreflang <a href='http://www.themediaflow.com/tool_hreflang' target='_blank'>hreflang XML sitemap generator</a>. You can feed it a CSV with URLs per language and it creates an XML sitemap. A very good first step when you decide to go this route.

The CSV file you feed this XML sitemap generator needs columns for every language. If you want to add an <code>x-default</code> URL to it as well, just create a column called <code>x-default</code>.
<h3>hreflang tag validator</h3>
<section class='alignright extra' id=''><p><a href='http://flang.dejanseo.com.au/' target='_blank'><img class='alignnone size-large wp-image-675748' src='https://yoast-mercury.s3.amazonaws.com/uploads/2016/03/flang-hreflang-validator-tool-600x208.png' alt='flang - hreflang validator tool by Dejan SEO' width='600' height='208'></img></a></p>
</section>

Once you've added markup to your pages, you'll want to validate that markup. If you choose to go the link element in the <head> route, you're in luck, as there are a few validator tools out there. The best one we could find is <a href='http://flang.dejanseo.com.au/' target='_blank'>flang, by DejanSEO</a>.

Unfortunately, we haven't found a validator for XML sitemaps yet.
<h2 id='hreflang-process'>Keeping hreflang alive: process</h2>
Once you've created a working hreflang setup, you need to set up processes. It's probably also a good idea to regularly audit your implementation to make sure it's still set up correctly.

Make sure that people in your company who deal with content on your site know about hreflang. This makes sure they won't do things that break your implementation. Two things are very important:
<ol>
 	<li>When a page is deleted, check whether its counterparts are updated.</li>
 	<li>When a page is redirected, change the hreflang URLs on its counterparts.</li>
</ol>
If you do that and audit regularly, you shouldn't run into many issues.
<h2>Conclusion</h2>
Setting up hreflang is a cumbersome process. It's a tough standard with a lot of specific things you should know and deal with. This guide will be updated as new things are introduced around this specification and best practices evolve, so check back when you're working on your implementation again!

<p class='readmore'><a title='rel=canonical: what is it and how (not) to use it' data-prefix='Read more' href='https://yoast.com/rel-canonical/'>Read more: ‘rel=canonical: what is it and how (not) to use it’ »</a></p><img width='266' height='139' src='https://yoast-mercury.s3.amazonaws.com/uploads/2016/04/9_Use_Hreflang_FI-1.png' class='attachment-266x266 size-266x266' alt='' srcset='https://yoast-mercury.s3.amazonaws.com/uploads/2016/04/9_Use_Hreflang_FI-1.png 1200w, https://yoast-mercury.s3.amazonaws.com/uploads/2016/04/9_Use_Hreflang_FI-1-250x131.png 250w, https://yoast-mercury.s3.amazonaws.com/uploads/2016/04/9_Use_Hreflang_FI-1-768x402.png 768w, https://yoast-mercury.s3.amazonaws.com/uploads/2016/04/9_Use_Hreflang_FI-1-600x314.png 600w' sizes='(max-width: 266px) 100vw, 266px'></img>
```

**Comment by iamazik** [comment]
The error also occurs with the following content which breaks the prominent words section:

```html
What are digital nomads, that must be the one question I get asked more than any other one. Since I'm working on a documentary where digital nomads play an essential role, here's a few parts of my answer:
<!--more-->
<h2>What are digital nomads?</h2>
Before we jump into the arguments, here’s my slightly broad definition of a digital nomad: <b>A digital nomad is a person who can live and work location independently, enabled or supported by information technology.</b>
<h2>History</h2>
<h3>The new way of working</h3>
Between 20 and 15 years ago, we started working from a technology perspective to what we then called the "<em>new way of working</em>." We started struggling with ideas around how "information workers" as we called them at that point, would organize and perform their work. The difference between the way an industrial job was executed, and a knowledge worker job, was the core driver of a lot of technology changes. Many of those we take for granted today, but they were small revolutions at that time: having personal email, a (portable) personal computer,

That was, without us having a clue at that point, the base that would serve as the enabling of location-independent work. One of the core principles we focused on, was the individual tasks of users. Where organizations before would have pools of staff doing all the same work, we were evolving towards more and more specialists and individuals doing one task in the whole complex system.
<h3>Flex Desk</h3>
Once we understood this new way or working, companies found a new way to save on operating costs. Paying rent and maintenance for an unmanned desk was seen as not cost effective. Companies decided to provide enough desks as the amount of staff they expected to be in the office at any given time. The team would find a spot to work on, but not have a private desk or office anymore.
<h3>Home working and "Home working."</h3>
And with the more natural ways to build VPN's over the internet, came the next step. If my employer had no desk anymore and took into account that I might be as creative from my own home than I would be in the office, companies started experimenting with home working.
<h3>Remote work.</h3>
Once we allowed and enabled home working, smarter individuals soon found out that they didn’t have to be at home, but could work anywhere. And see: there's the foundation for location independent working. Not so much a revolution, but an evolution of the past 20+ years. And here's the first step into knowing what are digital nomads.
<h2>Subgroups</h2>
<img class="alignleft wp-image-8921" src="https://boarding.today/wp-content/uploads/2018/03/20170416-2017-04-16-15.54.19-Airplane-Airport-ATL-Atlanta-avion-Delta-Air-Lines-Digital-Nomad-Laptop-Lounge-plane-Sky-Lounge-Vliegtuig-by-Koen-Blanquart-Boarding-Today-169x300.jpg" alt="What are digital nomads - Atlanta Airport working" width="150" height="266" />The New Nomads are not a homogeneous group. During my research, I found several types of movements. The one thing they have in common, being digital nomads, is that the information age allows them flexible time and location when it comes to working. Amongst the Digital Nomads, multiple groups can be identified :
<ol>
 	<li>The <b>location-independent entrepreneurs. </b>&lt;/ br&gt;
These people make enough money in this new economy to be fully independent. They can decide to live in New York, or in a country with a lower cost of living. Some of them travel between their “own” places.</li>
 	<li>A second (large) group is what we, for now, call "<b>backpackers with a laptop"</b>
They travel around the world and make (just) enough money to keep them going. They are location independent for as much of their budget allows them. Many of them travel Southeast Asia. Thailand, Laos, and especially Bali host communities of these digital nomads, backpackers with a laptop.</li>
 	<li>The third group is people who combine their working in the old/industrial model with traveling but at separate times. They <b>work-save-travel-repeat</b>. Some of them work for information age companies that do not support the nomad idea, so they are limited to this form of digital nomadism.</li>
</ol>
&nbsp;
<h2>Are all Digital Nomads freelancers?</h2>
When I was preparing for a keynote speech with an employers organisation in New York recently, their executive director had an open conversation with me, expressing worries that I would promote the nomad lifestyle to the audience, and make them all freelancers that would roam the planet, thereby taking them away from the workforce available to their members.

But I assured them that location independent work does NOT mean employees massively quit the company. But it is a fact that the relationship between employers and employees has been shifting over the past decade. The individual knowledge that an information worker brings to the company shifts its hiring power. But I see many organizations embracing the location independence of their teams, sometimes encouraging them to investigate the nomadic lifestyle.
And not all employees are willing or ready not to have a water cooler, an office to go to, and some structure and routine. It feels like we’re heading to a world where location-independence will become ever more present, while many organizations still maintain facilities where the employees can come “home” and feel part of the culture.

When seekingto understand what are digital nomads, we must ook at the way they work, not the relation they have with their employer/client as the determening factor.
<h2><img class="alignright size-medium wp-image-8926" src="https://boarding.today/wp-content/uploads/2018/03/20160914-2016-09-14-06.40.23-Dropbox_camera_uploads-Saint-Petersburg-SaintPetersBurg-Moscow-Sapsan-train-by-Koen-Blanquart-Boarding-Today-300x169.jpg" alt="" width="300" height="169" />What are digital nomads locations?</h2>
I've not found a space in my travels of the last years where I didn't encounter some nomads. But a vast majority ends up in South East Asia. Bali and Thailand are amongst the more popular regions where they locate. Certain places, such as Chian Mai are hubs for specialties even. Many marketing specialists and social media consultants are grouping here.

There are probably no spots on the planet where no digital nomad could be found one day. But a majority of them is drawn to the same places. A few of the hot spots today include:
<ul>
 	<li>For Asia: In <a href="https://boarding.today/destination/thailand/">Thailand</a>, many of them in the Chian Mai area and in <a href="https://boarding.today/?s=Bangkok">Bangkok</a>; In Indonesia, around <a href="/?s=ubud">Ubud</a> (Bali)</li>
 	<li>When you look around in Europe, you'll find that Barcelona (Spain) and Lisbon (Portugal) are pretty popular destinations. The <a href="https://boarding.today/visit-vilnius-lithuania/">Baltic states</a>, with Vilnius as the most popular one, are being discovered by remote workers.</li>
 	<li>In South America, <a href="https://boarding.today/th%d1%96ng%d1%95-to-do-when-v%d1%96%d1%95%d1%96t%d1%96ng-buenos-aires/">Buenos Aires </a>is a hit, and Medellin in Colombia attracts both groups and individual working travelers;</li>
 	<li>Africa is a bit underrepresented, but I seem to meet these days more location independent workers who are evaluating to spend some time in Africa. I received offers from Ethiopia, ... and ... in the past months, to come and check out places built for traveling information workers.</li>
</ul>
There's a crowdsourced list of places where nomads collect their experience with places and can find inspiration for the next location, called "Nomadlist." (by <a href="https://twitter.com/levelsio">Pieter Levels</a>).

<img class="aligncenter size-large wp-image-8927" src="https://boarding.today/wp-content/uploads/2018/03/20170813-2017-08-13-09.15.18-Airplane-Airport-ATL-Atlanta-avion-plane-Vliegtuig-by-Koen-Blanquart-Boarding-Today-1024x514.jpg" alt="A Delta Air Lines Plane ready to take off in Atlanta Airport - Image by Koen Blanquart" width="1024" height="514" />
<h2>What is a digital nomad's ideal workspace?</h2>
These travelers, of course, require organizing their work. While they technically can work from anywhere, you'll find many of them choosing one of these options:
<h3>Working in a coworking space</h3>
This might sound like the most obvious choice. And in many locations, nomads do indeed find coworking spaces to get work done. However, many of them feel this as too corporate and still seek other solutions. Some companies such as <a href="https://www.wework.com/" rel="nofollow noopener">WeWork</a>, are betting heavily on the nomads using workspaces and they even invest in programs that coach new nomads, such as <a href="https://remoteyear.com/" rel="noopener">Remoteyear</a>.
<h3>Hotels &amp; BnB's</h3>
When traveling, working from a hotel room or a BnB is often the way to go. This article has partially been prepared in a co-working space in Thailand but was edited in the common area of an Airbnb in <a href="/destination/panama/">Panama</a>. It's easy to start or end the day with the laptop or phone connected.

Airbnb, for example, understood the importance hereof when they added the "<a href="https://airhostsforum.com/t/laptop-friendly-workspace/17530" rel="nofollow noopener">Laptop friendly workspace</a>" in the amenities of a place.
<h3>Coffeeshops</h3>
[caption id="attachment_8919" align="alignleft" width="300"]<img class="size-medium wp-image-8919" src="https://boarding.today/wp-content/uploads/2018/03/20160906-2016-09-06-09.40.05-Cologne-Dropbox_camera_uploads-Germany-by-Koen-Blanquart-Boarding-Today-300x169.jpg" alt="Working in a coffeeshop as digital nomad" width="300" height="169" /> Working in a coffee shop as a digital nomad[/caption]

Coffeeshops have four of the elements many nomads are seeking in a place to do some work:
<ol>
 	<li>Coffee, ( or tea, juice, you see where this is going) and often some original food.</li>
 	<li>(Free) internet, so we can <s>update <a href="https://instagram.com/kblanqua/" rel="noopener">our Instagram</a></s> get some work done</li>
 	<li>A bathroom, cause we're all humans</li>
 	<li>Other Nomads or patrons to have a conversation with</li>
</ol>
But the choices are endless. In countries where the mobile internet is affordable and reliable, there's no limit to where to get some work done.

[caption id="attachment_8907" align="alignright" width="193"]<img class="wp-image-8907" src="https://boarding.today/wp-content/uploads/2018/03/20160911-HQ3A0118-Koen-Blanquart-Lithuania-Vilnius-by-Koen-Blanquart-Boarding-Today-300x300.jpg" alt="Koen Blanquart writing" width="193" height="193" /> Koen Blanquart writing[/caption]
<h2>Koen Blanquart, Boarding Today and Digital Nomads.</h2>
<a href="https://boarding.today/contactkoen/">Koen Blanquart</a> is working on <a href="https://boarding.today/digital-nomads-location-independent-living/">a documentary about the shift in society, focusing on digital nomads and other location independent lifestyles</a>. Some of the findings can be found on the <a href="https://boarding.today/">Boarding Today</a> blog or ar part of<a href="https://boarding.today/koen-blanquart-public-speaker/"> the speeches he gives</a>.

This article "what are digital nomads" is only one article from a series of location-independent work I'm researching. We'll be working further on the question "what are digital nomads" after the first take since we're gathering more information for our <a href="https://boarding.today/digital-nomads-location-independent-living/">articles about digital nomads and location independent living and working </a>and our documentary.

If you are a location independent professional with a unique story: get in touch! We seeking digital nomads who'd like to share their story here, or in the upcoming documentary. <strong><a href="https://boarding.today/contactkoen/">Contact Koen</a></strong> if you like to discuss.
```

**Comment by benvaassen** [comment]
This should be fixed with 9.2 because it's a duplicate of issue https://github.com/Yoast/YoastSEO.js/issues/840
